tad/site/templates/default_layout.jinja

```diff
@@ -0,0 +1,90 @@
+{% macro render_task_card(task) -%}
+    {% include "task.jinja" %}
+{%- endmacro -%}
+
+{% macro render_task_column(status) -%}
+    <div class="col span_1_of_4">
+        <h3 class="text-center-horizontal margin-bottom--small">{{ status.name }}</h3>
+        <div class="progress_cards_container sortable" data-id="{{ status.id }}" id="column-{{ status.id }}">
+            {% for task in tasks_service.get_tasks(status.id) %}
+                {{ render_task_card(task) }}
+            {% endfor %}
+        </div>
+    </div>
+{% endmacro -%}
+
+<!DOCTYPE html>
```

Comment:
Could you split up the html in a bit more parts

Reply:
I will leave this for the next iteration, as the frontend probably will change a lot.